docs/contributing/contributing-code/contributing-code-first-commit/first-commit-04-debugging-cli/index.md

```diff
@@ -2,67 +2,76 @@
 
 ## Debugging your changes
 
-### Creating debug settings
```

Comment:
I previewed this in GitHub to verify that all of the images still work. Looks good 👍